test/uk/gov/hmrc/excisemovementcontrolsystemapi/connector/util/EISHttpReaderSpec.scala

```diff
@@ -17,22 +17,32 @@
 package uk.gov.hmrc.excisemovementcontrolsystemapi.connector.util
 
 
+import akka.http.scaladsl.model.DateTime
```

Comment:
Do we want to be using this import? We already have LocalDateTime floating about here

Reply:
nop, we don`t. Removed